### PR TITLE
Initial mock-server

### PR DESCRIPTION
**Pull request by brylie**
We agreed to provide a proof-of-concept server to provide mock data that can be used by developers. This PR is the initial effort to set up the mock server.

### CODE REVIEW

**Comment by brylie** [comment]
I am having difficulty getting past the auth stage. How should we set up or disable authentication for this mock-server?

**Comment by nighthawk** [comment]
You could use some kind of simple pseudo-authentication. Say, you have endpoints that require a "login token" for the user to be passed as a header. Your mock server could then still require that but it just checks it against a hard-coded valid token, e.g., "X-MaaS-User-Token: test-token".

**Comment by brylie** [comment]
Here is the security configuration for the Bookings API:

https://github.com/maas-alliance/apis/blob/4e312581989c7a69e7ef194f357c1b2f2b68d070/specifications/booking.yaml#L31-L32

So, it looks like you recommend using this approach?
https://connexion.readthedocs.io/en/latest/security.html#apikey-authentication

**Comment by brylie** [comment]
@nighthawk I would like to keep the scope of this PR relatively small, so we can work in parallel on mock-data functions. Would you mind reviewing and, if permissible, signing-off on this PR?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 # apis
 Example APIs designed to support the MaaS ecosystem.
+
+## Running the server
+To run this server, use the following steps:
+
+1. clone this repository
+2. set up a virtual environment
+3. install the `requirements.txt`
```
Comment (nighthawk) [inline README.md]: Not too familiar with Python package managers. Is there a terminal command for this?

File: app.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import connexion
```
Comment (nighthawk) [inline app.py]: Should we move this to `mock_server/app.py`?

File: api.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+def bookings_options_get(message):
```
Comment (nighthawk) [inline api.py]: I suggest to move this file to something like `mock_server/api.py`

File: app.py (hunk 2 above)
Comment (brylie) [reply app.py]: Done.

File: README.md (hunk 1 above)
Comment (brylie) [reply README.md]: Added a link to Python 3 documentation.

File: api.py (hunk 3 above)
Comment (brylie) [reply api.py]: Done.

**Pull request merged**